python/lsst/ts/ofc/scripts/generate_sensitivity_matrix.py

```diff
@@ -0,0 +1,393 @@
+# This file is part of ts_ofc.
+#
+# Developed for Vera Rubin Observatory.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import argparse
+import pathlib
+import textwrap
+from typing import Tuple
+
+import batoid
+import batoid_rubin
+import galsim
+import numpy as np
+import ruamel.yaml
+from astropy.io import fits
+from tqdm import tqdm
+
+
+def get_fiducial(
+    instrument: str,
+    band: str,
+) -> Tuple[batoid.Optic, float]:
+    """Get fiducial optical system and wavelength
+    for a given instrument and band.
+
+    Parameters
+    ----------
+    instrument : str
+        Name of the instrument. Either 'LSST' or 'ComCam'.
+    band : str
+        Name of the filter. Either '', 'u', 'g', 'r', 'i', 'z', or 'y'.
+
+    Returns
+    -------
+    fiducial : batoid.Optic
+        Fiducial optical system.
+    wavelength : float
+        Wavelength in um.
+    """
+
+    # Get fiducial optical system from batoid.
+    if band == "":
+        fiducial = batoid.Optic.fromYaml(f"{instrument}_g_500.yaml")
+    else:
+        fiducial = batoid.Optic.fromYaml(f"{instrument}_{band}.yaml")
+
+    # Get corresponding filter wavelength in um.
+    wavelength = {
+        "U": 0.365,
+        "G": 0.480,
+        "R": 0.622,
+        "I": 0.754,
+        "Z": 0.869,
+        "Y": 0.971,
+        "": 0.5,
```

Comment:
This part should belong to a configuration file.

Reply:
Done! Added in config files